packages/react-art/src/ReactARTHostConfig.js

```diff
@@ -9,13 +9,15 @@ import * as ReactScheduler from 'shared/ReactScheduler';
 import Transform from 'art/core/transform';
 import Mode from 'art/modes/current';
 import invariant from 'fbjs/lib/invariant';
-import emptyObject from 'fbjs/lib/emptyObject';
 
 import {TYPES, EVENT_TYPES, childrenAsString} from './ReactARTInternals';
 
 const pooledTransform = new Transform();
 
-const UPDATE_SIGNAL = {};
+const emptyObject = {};
```

Comment:
Why go from a descript explicit identity to a non-descript, easy to accidentally share object?

Reply:
I was on the fence about this. Even if it's shared, does it matter if it's frozen in DEV? Seems like you'd quickly catch the mistake. It's also used under this name for context. Do you want to have a separate `EMPTY_CONTEXT` too? I can revert that part.

Reply:
Pushed https://github.com/facebook/react/pull/13055/commits/1f0c930aae723f4a066a8751d1264473bd95b43f which makes explicit naming consistent across renderers